index.html

```diff
@@ -197,10 +202,47 @@ <h3>Skills</h3>
             <li>Codepen</li>
             <li>Gitlab</li>
             <li>Terminal</li>
-          </ul>  
+          </ul>
         </div>
       </li>
     </ul>
   </section>
+
+  <section class="contact">
+    <p class="get-touch">
+      I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+    </p>
+    <form class="data-form" action="https://formspree.io/f/mjvlbpay" method="post">
+      <label for="contact-name">
+        <input type="text" class="contact-form" id="contact-name" name="contact-name" placeholder="Full name" maxlength="30" required>
+      </label>
+      <label for="contact-email">
+        <input type="email" class="contact-form" id="contact-email" name="contact-email" placeholder="Email address" required>
+      </label>
```

Comment:
- [ ] Buy following the requirements and the suggestion of the previous reviewer this form has a background image. check the image bellow :point_down:
![Screenshot from 2021-12-01 01-47-20](https://user-images.githubusercontent.com/65068771/144147284-37ea3ac5-3dbe-4420-808e-391f88494f1f.png)

- [ ]  Kindly let's go ahead and add that background image.